src/borg/helpers.py

```diff
@@ -1527,12 +1531,21 @@ def get_item_data(self, item):
             item_data[key] = self.call_keys[key](item)
         return item_data
 
-    def calculate_num_chunks(self, item):
-        return len(item.get('chunks', []))
+    def sum_unique_chunks_property(self, property_func, item):
+        """
+        sum unique chunks properties, a unique chunk is a chunk which is referenced globally as often as it is in the
+        item
 
-    def calculate_unique_chunks(self, item):
+        item: The item to sum it's unique chunks
```

Comment:
>  (here we see where the confusion comes from: possessive form for nouns is with apostrophe, but the possessive pronoun "its" is without ...)

I wondered why I kept doing this specific mistake -- I don't think I make any other error as frequently as this one, normally I even get their/they're/there right. This is an interesting explanation. Not only does its/it's just not exist in my first language, but it's also kind of inconsistent.